vllm/model_executor/models/transformers.py

```diff
@@ -204,36 +300,55 @@ def apply_base_model_tp_plan(self, module: nn.Module, prefix: str = ""):
             else:
                 self.apply_base_model_tp_plan(child_module, prefix=qual_name)
 
-    def replace_vocab_embed_class(self, module: nn.Module):
-        # Use native set input embeddings
-        new_module = VocabParallelEmbedding(
-            self.vocab_size,
-            self.config.hidden_size,
-            org_num_embeddings=self.config.vocab_size,
-            quant_config=None,
-        )
-        log_replacement("input embedding", self.model.get_input_embeddings(),
-                        new_module)
-        self.model.set_input_embeddings(new_module)
+    def init_buffers(self, module: nn.Module):
```

Comment:
General comment, might it be better to move this function to a separate transformers_utils file?

Reply:
We could, but I worry that would pollute the `models` directory